Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridDataConnection.cs

```diff
@@ -27,7 +28,9 @@ internal class DataGridDataConnection
         private IEnumerable _dataSource;
         private Type _dataType;
         private bool _expectingCurrentChanged;
+        private ISupportIncrementalLoading _incrementalItemsSource;
         private object _itemToSelectOnCurrentChanged;
+        private IAsyncOperation<LoadMoreItemsResult> _loadingOperation;
         private DataGrid _owner;
         private bool _scrollForCurrentChanged;
         private DataGridSelectionAction _selectionActionForCurrentChanged;
```

Comment:
Also declare below here, you'll see why:
`private WeakEventListener<DataGridDataConnection, object, PropertyChangedEventArgs> _weakIncrementalItemsSourcePropertyChangedListener;`